content/docs/policy/approval/approver-policy/README.md

```diff
@@ -271,6 +271,43 @@ spec:
   ...
 ```
 
+#### Validations
+
+Since approver-policy 0.11.0 it is now possible to define more advanced allowed
+request attribute validation rules using
+[Common Expression Language (CEL)](https://kubernetes.io/docs/reference/using-api/cel/).
+The main motivation for adding this feature, was to enable allowed request
+attribute values as a function of the request namespace. But since CEL is a
+small programming language, validation rules can also complement or replace the
+basic wildcard support in allowed attribute value specification.
+
+The request attribute value is made available to the expression in the `self` variable.
+For multi-valued request attributes, the validation will be performed once per value.
+Similarly to the `self` variable, approver-policy provides the `cr` variable representing
+the request to validate. This variable is of object type, and at time of writing it has
+just two fields: `namespace` and `name`.
+
+In the following, we use approver-policy CEL validation rules to ensure
+[X.509 SVID certificates](https://github.com/spiffe/spiffe/blob/main/standards/X509-SVID.md)
+are issued with a [SPIFFE ID](https://github.com/spiffe/spiffe/blob/main/standards/SPIFFE-ID.md)
+(X.509 URI SAN) identifying the namespace (and service account).
+
+```yaml
+spec:
+  ...
+  allowed:
+    uris:
+      validations:
+      - rule: self.startsWith('spiffe://trust.domain/ns/' + cr.namespace + '/sa/')
+        message: only URIs representing the current namespace in the SPIFFE ID are allowed
```

Comment:
It's the recommended practice to not have a trailing period?

Reply:
I am unsure, but I think a trailing period could make sense. I'll update it.